pkg/logs/input/tailer/scanner.go

```diff
@@ -40,11 +43,13 @@ func New(sources []*config.LogSource, tailingLimit int, pp pipeline.Provider, au
 		}
 	}
 	return &Scanner{
-		pp:           pp,
-		tailingLimit: tailingLimit,
-		fileProvider: NewFileProvider(tailSources, tailingLimit),
-		tailers:      make(map[string]*Tailer),
-		auditor:      auditor,
+		pp:                  pp,
+		tailingLimit:        tailingLimit,
+		fileProvider:        NewFileProvider(tailSources, tailingLimit),
+		tailers:             make(map[string]*Tailer),
+		auditor:             auditor,
+		tailerSleepDuration: tailerSleepDuration,
+		done:                make(chan struct{}),
```

Comment:
set size to 1?

Reply:
Same here, we want to avoid stopping all the tailers while scanning at the same time.